packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testSourceCodeWhenMethodDoesNotExistAnymore.st

```diff
@@ -0,0 +1,22 @@
+as yet unclassified
+testSourceCodeWhenMethodDoesNotExistAnymore
+
+	| messageTallyMock compiledMethod tree source |
+	
+	compiledMethod := (DummyObject>>#dummyMethodToRemove).
+	
+	messageTallyMock := MessageTallyMock new.
+	messageTallyMock method: compiledMethod.
+	 
+	tree := IPTTreeModelMock new.
+	tree currentSelection: messageTallyMock.
```

Comment:
Lines 8 to 12 are duplicates of `IPTTreeModelTestCase >> testSourceCodeWhenClassDoesNotExistAnymore` lines 8 to 12. Please take appropriate action to avoid code duplication.